packages/@uppy/provider-views/src/index.js

```diff
@@ -420,7 +420,7 @@ module.exports = class ProviderView {
   handleAuth () {
     const authState = btoa(JSON.stringify({ origin: getOrigin() }))
     // @todo remove this hardcoded version
-    const clientVersion = 'companion-client:1.0.2'
+    const clientVersion = encodeURIComponent('@uppy/companion-client=1.0.2')
```

Comment:
`ProviderView` just got `VERSION` too.

Reply:
added an asana task to address these in a followup, marking comments as resolved here